### ISSUE TITLE
The document root directory "./MyTestPlugin/public" does not exist.

### ISSUE THREAD

**Issue by doctorx32** [Sylius/PluginSkeleton#33]
I'm trying to test app in my browser, as written in Readme.md, but receive this error:
```
MyTestPlugin\tests\Application> php bin\console server:run -e test

 [ERROR] The document root directory "C:\Coding\php\MyTestPlugin/public" does not exist.
```

**Comment by waqarmuneer** [comment]
I got the same issue and solved it by adding docroot in the command. Like:
`php bin/console server:run --env=dev --docroot=/path/to/MyPlug/tests/Application/web`

**Comment by pamil** [comment]
`--docroot=web` / `-d web` is enough, fixed on Travis by #39.

**Comment by bitbager** [comment]
It would be nice to mention it in the README.md.
